docs/usage/models.md

```diff
@@ -451,6 +459,23 @@ of the data provided.
 For example, in the example above, if `_fields_set` was not provided,
 `new_user.model_fields_set` would be `{'id', 'age', 'name'}`.
 
+Note that for subclasses of [`RootModel`](#custom-root-types), the root value can be passed to `model_construct`
+positionally, instead of using a keyword argument.
+
+Here are some additional notes on the behavior of `model_construct`:
+* When we say "no validation is performed" — this includes converting dicts to model instances. So if you have a field
+  with a `Model` type, you will need to convert the inner dict to a model yourself before passing it to
+  `model_construct`.
+  * In particular, the `model_construct` method does not support recursively constructing models from dicts.
+* If you do not pass keyword arguments for fields with defaults, the default values will still be used.
+* For models with `model_config['extra'] == 'allow'`, data not corresponding to fields will be correctly stored in
+  the `__pydantic_extra__` dict.
+* For models with private attributes, the `__pydantic_private__` dict will be initialized the same as it would be when
+  calling `__init__`.
+* When constructing an instance using `model_construct()`, no `__init__` method from the model or any of its parent
+  classes will be called, even when a custom `__init__` method is defined.
```

Comment:
This is a lot of information - not a helpful comment, but that's what was in my mind.